src/main/java/com/example/Storehouse/service/StorehouseService.java

```diff
@@ -0,0 +1,80 @@
+package com.example.Storehouse.service;
+
+import com.example.Storehouse.builder.refund.RefundResponseBuilder;
+import com.example.Storehouse.entity.Products;
+import com.example.Storehouse.models.product.Product;
+import com.example.Storehouse.models.request.Refund;
+import com.example.Storehouse.models.request.RequestMultiProducts;
+import com.example.Storehouse.builder.multiproduct.MultiProductResponseBuilder;
+import com.example.Storehouse.models.response.multiProductsResponse.MultiProductsResponse;
+import com.example.Storehouse.builder.quantity.QuantityOfProductResponseBuilder;
+import com.example.Storehouse.models.response.quantityResponse.QuantityResponse;
+import com.example.Storehouse.builder.singleProduct.SingleProductResponseBuilder;
+import com.example.Storehouse.models.response.singleProductResponse.SingleProductResponse;
+import com.example.Storehouse.repository.ProductRepository;
+import com.example.Storehouse.logger.Print;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class StorehouseService implements iStorehouseService {
+    @Autowired
+    private SingleProductResponseBuilder responseBuilding;
```

Comment:
Do not Autowire any models, its not a component that has logic, it just a model
Instead take a look at lombok @Builder annotation

Reply:
done